sources/tools/Stride.TextureConverter/Backend/Wrappers/FINetWrapper/FreeImageWrapper.cs

```diff
@@ -2060,7 +2060,7 @@ public static FIMULTIBITMAP OpenMultiBitmapFromStream(Stream stream, ref FREE_IM
 				if (!mdib.IsNull)
 					CloseMultiBitmap(mdib, FREE_IMAGE_SAVE_FLAGS.DEFAULT);
 
-				if (handle != null)
+				if (handle.IsNull == false)
```

Comment:
Any reason why you're not using `if (!handle.IsNull)`?

Reply:
Nothing besides my own style, I don't tend to use it as I find that character quite easy to miss when skimming through logic